docs/guides/using-suites.md

```diff
@@ -0,0 +1,98 @@
+# Using suites
+
+With increasing number of MFTF tests, it's important to have a mechanism to organize and consolidate them for ease-of-use.
+
+## What is a suite?
+
+Suite is a collection of MFTF tests that are intended to test specific behaviors of Magento. It may contain common initialization and clean up steps specific to the test cases included. It allows you to include, exclude and/or group tests with preconditions and post conditions.
+You can create a suite referencing tests, test groups and modules.
+
+### How is a suite defined?
+
+Suite must be defined under `<magento 2 root>/dev/tests/acceptance/tests/_suite` as an xml file. The generated tests for each suite are grouped into their own directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/`
```

Comment:
Suites can also live under `<module>/Test/Mftf/Suite`

For example:
```
magento2ce/app/code/Magento/Paypal/Test/Mftf/Suite/suite.xml
```

Reply:
added.